libs/website/ui/sponsorship/sponsorship-presenter/sponsorship.presenter.tsx

```
@@ -40,17 +42,10 @@ export function SponsorshipPresenter({
           </Button>
         </Link>
       </div>
+      <div className="bg-border h-0.5 rounded w-[95%] m-auto"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)